config/common/errors.go

```diff
@@ -54,7 +54,8 @@ var (
 	ErrTooFewMirrorDevices     = errors.New("mirroring requires at least two devices")
 
 	// partition
-	ErrWrongPartitionNumber = errors.New("incorrect partition number; a new partition will be created using reserved label")
+	ErrInvalidPartitionConfig = errors.New("unexpected partition number. Partition number should only be zero when using root disk or wipe_tables is true. Specify root disk as /dev/disk/by-id/coreos-boot-disk ")
```

Comment:
Sorry not trying to bike-shed, wdyt of this ? => 
"unexpected partition number; zero is for root disk or when wipe_tables is set to true. To declare a root disk specify the config.disk.device as /dev/disk/by-id/coreos-boot-disk")"

Reply:
sounds good ill add it now!